### PR TITLE
Feature routes and user settings

### PR DESCRIPTION
**Pull request by shagorc1**
### Description:
- Added _user settings_ to **db.json**
- Created **routes.js** file to import routes
- Modified code to validate **POST**, **PUT**  or **DELETE**

### CODE REVIEW

File: routes.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const routes = {
+    '/user/settings': '/user_settings',
```
Comment (nancybarraza) [inline routes.js]: I would consider adding the complete service path, so we can replace it easier in the app.
`/users-service/user/settings`
What do you think?

File: server.js [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
 const middlewares = jsonServer.defaults();
 const port = process.env.PORT || 4000;
 
 server.use(middlewares);
+server.use(jsonServer.rewriter(routes));
+server.use(jsonServer.bodyParser);
+server.use((req, res, next) => {
+    // validate if request method is POST, PUT or DELETE and send direct status 200 success
+    // if catch not it, this clean the mock data with "{}" over property request
+    if (req.method === 'POST' || req.method === 'PUT' || req.method === 'DELETE') {
```
Comment (nancybarraza) [inline server.js]: add the patch request method

**Review by nancybarraza — CHANGES_REQUESTED** [review]

File: db.json [hunk 3]
```diff
@@ -12,5 +12,81 @@
       "content": "Wow such content!",
       "username": "zero_cool"
```
Comment (nancybarraza) [inline db.json]: Can you also remove the dummy data? Leave only the project mocks; thanks!

**Review by nancybarraza — CHANGES_REQUESTED** [review]

File: server.js (hunk 2 above)
Comment (shagorc1) [reply server.js]: **PATCH** request doesn't affect the mock data, that's why I didn't include it

Comment (nancybarraza) [reply server.js]: just add it because we will simulate the requests :P

File: routes.js (hunk 1 above)
Comment (shagorc1) [reply routes.js]: done

File: server.js (hunk 2 above)
Comment (shagorc1) [reply server.js]: done

File: db.json (hunk 3 above)
Comment (shagorc1) [reply db.json]: done

**Review by nancybarraza — APPROVED** [review]

**Pull request merged**